framework/auth/exceptions.py

```diff
@@ -60,3 +60,27 @@ class ExpiredTokenError(EmailConfirmTokenError):
     """Raised if an email confirmation token is expired."""
     message_short = "Expired Token"
     message_long = language.EXPIRED_EMAIL_CONFIRM_TOKEN
+
+
+class MergeConfirmedRequiredError(EmailConfirmTokenError):
+    """Raised if a merge is possible, but requires user confirmation"""
+
+    def __init__(self, user, user_to_merge, *args, **kwargs):
```

Comment:
First arguent to an `Exception` first argument should be its message so that tracebacks will display correctly.